testing-guidance/workflow-tests/shapes/shapes-lkg-workflows.md

```diff
@@ -0,0 +1,71 @@
+# Shapes LKG Workflow Tests
+
```

Comment:
It'd be nice to have a list of the shapes components first, I don't see that list explicitly in the document at the moment, only dependencies to them.

Reply:
The list is in the docs I linked, and easy to discover in the engine. I don't think it adds a lot to duplicate it here and just adds maintenance burden if any new shapes get created.

Reply:
I'd love to hear @FuzzyCarterAWS feedback on this. Other LKGs I think they do list the components. It's true that they can get out of date, but the doc is meant to be updated if things change. Not mentioning the components can equality end up with some being missed and not tested as well.

Reply:
In this context, since we have a really good Docs page for this (https://www.o3de.org/docs/user-guide/components/reference/shape/), we can use the documentation.

However, we should reference this doc in the workflow as "supported shape components."